DataFormats/MuonReco/interface/Muon.h

```diff
@@ -219,7 +219,9 @@ namespace reco {
       MultiIsoMedium         = 1UL<<26,   // miniIso with ptRatio and ptRel 
       PuppiIsoLoose          = 1UL<<27,
       PuppiIsoMedium         = 1UL<<28,
-      PuppiIsoTight          = 1UL<<29
+      PuppiIsoTight          = 1UL<<29, 
+      MvaVTight              = 1UL<<30,   // > 0.45
+      MvaVVTight             = 1UL<<31,   // > 0.9
```

Comment:
as we are nearing an end of the ```unsigned int``` range, I have a question where does this go next.
This enum is already using long unsigned int  (```UL```).
One option is to promote the ```selectors_``` data member to an unsigned long int.
This is a comment outside of the scope of this PR at the moment.